pkg/grizzly/config.go

```diff
@@ -2,8 +2,8 @@ package grizzly
 
 // GrizzlyOpts contains options for all Grizzly commands
 type GrizzlyOpts struct {
-	Directory    *string
-	ResourceFile *string
+	ResourcePath *string
```

Comment:
ResourcePath appears mandatory (it can't work without, right?) It shouldn't be in `opts` then, let's leverage the fact that function arguments are mandatory as well and compile-time checked